### PR TITLE
Next steps

### PR DESCRIPTION
**Pull request by edewit**
introduce "Next steps" "page" with links to console and github

### CODE REVIEW

File: src/app/wizard/pages/nextSteps/nextSteps.page.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <div class="col-md-9">
+      <p>
+        <a [href]="consoleUrl()">Take a look at your CI/CD build <i class="fa fa-external-link" aria-hidden="true"></i></a><br/>
+        <i class="text-center">This will take a couple of minutes</i>
+      </p>
+    </div>
+  </div>
+  <div class="row primary">
+    <div class="col-md-3 primary">
+      Your next Step
```
Comment (gastaldi) [inline src/app/wizard/pages/nextSteps/nextSteps.page.html]: `Your next step` reads better (lowercase 's')

**Comment by gastaldi** [comment]
Can we use the real values in the `git clone` command? 

screenshot from 2017-07-03 18-48-16

**Comment by gastaldi** [comment]
Also maybe we should use HTTPS instead of SSH in the `git clone` command, for Windows users

**Comment by gastaldi** [comment]
The `Take a look at your CI/CD build` link doesn't work

**Comment by edewit** [comment]
it already puts the url into the documentation, but for that to work the documentation PR needs to be merged: https://github.com/openshiftio/appdev-documentation/pull/324

**Comment by edewit** [comment]
Commit 1d6bdad fixes the CI/CD link

**Comment by edewit** [comment]
@gastaldi okay to merge?

**Comment by gastaldi** [comment]
Looks good, but I wonder if we could use the real values in the Git clone URL, as I mentioned in https://github.com/openshiftio/launchpad-frontend/pull/143#issuecomment-312739307

**Comment by gastaldi** [comment]
Also I don't think we should allow navigating back to the previous steps once finished.

**Comment by edewit** [comment]
screenshot from 2017-07-11 08-51-56
It does fill in the real git clone URL now that the Doc PR is updated...

**Pull request merged**